include/net/socket.h

```diff
@@ -208,6 +208,14 @@ __syscall int z_zsock_getaddrinfo_internal(const char *host,
 					   const struct zsock_addrinfo *hints,
 					   struct zsock_addrinfo *res);
 
+#define AI_PASSIVE 1
```

Comment:
Why do we keep mixing decimal/hex values in this header. Can't we be consistent here and write all flags in hex?

Reply:
> Why do we keep mixing decimal/hex values in this header. Can't we be consistent here and write all flags in hex?

I guess, we're optimizing for size and ease of writing ;-). As said, this headers "keeps" doing that for a while, and now it's consistent to just do that. Of course, we may want to upgrade it all consistently later. In the meantime, it's not incorrect, and I can testify that I've written every current there myself, and didn't copied from some 3rd-party library, which may have implications.

Reply:
I'm not claiming it's incorrect nor suspect you for some blind copy-pasting. Just trying to understand the reasoning behind it, I don't find it a big deal. All in all, machine in the end doesn't care.

Reply:
> nor suspect you for some blind copy-pasting

It's good that you don't. But even if somebody does, they would have hard time to find an expert who would try to prove otherwise (and wouldn't lie). 

Any other reasoning you would like to discuss?

Reply:
Well we can go further and ask why `0x10` and not `16` if we optimize for size, but I guess that's where "ease of writing" part comes in. But honestly - I don't think it's worth a discussion.

Reply:
Looks indeed very ugly. We could just fix these to be hex in the first place, this should not take long to fix.

Reply:
Ok, added "0x".